common/rya.api/src/main/java/org/apache/rya/api/instance/ConfigurationFields.java

```diff
@@ -31,5 +31,7 @@
     static final String USE_FREETEXT = "sc.use_freetext";
     static final String USE_TEMPORAL = "sc.use_temporal";
     static final String USE_ENTITY = "sc.use_entity";
-    static final String USE_PCJ = "sc.use_pcj";
```

Comment:
By getting rid of this flag, can you no longer create a client that is able to disable the PCJ secondary index when said index exists?